apps/wallet/src/ui/app/components/active-coins-card/CoinItem.tsx

```diff
@@ -15,14 +15,15 @@ type CoinItemProps = {
 
 export function CoinItem({ coinType, balance, isActive, usd }: CoinItemProps) {
     const [formatted, symbol] = useFormatCoin(balance, coinType);
+    const { data: metadata } = useCoinMetadata(coinType);
 
     return (
         <div className="flex gap-2.5 w-full justify-center items-center">
             <CoinIcon coinType={coinType} size={isActive ? 'sm' : 'md'} />
             <div className="flex flex-1 gap-1.5 justify-between">
                 <div className="flex flex-col gap-1.5">
                     <Text variant="body" color="steel-darker" weight="semibold">
-                        {symbol} {isActive ? 'available' : ''}
+                        {metadata?.name || symbol} {isActive ? 'available' : ''}
```

Comment:
I’m not sure the name is the right thing to use, symbol is what’s generally used elsewhere, name is more for token lists.

Reply:
We probably want to start a conversation on slack with devx about the intended use of the name vs the symbol so that we can handle this consistently.

Reply:
Got it